src/content/about-the-data/subaward-data-quality.mdx

```diff
@@ -7,16 +7,15 @@ of Management and Budget (OMB) updated
  <a target="_blank" rel="noopener noreferrer" href="https://www.ecfr.gov/current/title-2/subtitle-A/chapter-I/part-170/subpart-A/section-170.105">2 CFR § 170.105</a>
  to expand the subaward reporting requirements to other types of financial
 assistance beyond grants and cooperative agreements, such as direct
-payments (direct appropriations), loans, subsidies, and insurance however, as of August 2022, FSRS has
+payments (direct appropriations), loans, subsidies, and insurance; however, as of August 2022, FSRS has
 not been updated to allow for reporting of these other assistance
 types.
 
 While federal agency personnel do not submit subaward data themselves,
 they are required to ensure that the award terms and conditions for
 any federal award include specific language requiring subaward
-reporting (subject to certain exceptions listed in
- <a target="_blank" rel="noopener noreferrer" href="https://www.ecfr.gov/current/title-2/subtitle-A/chapter-I/part-170/subpart-A/section-170.110">2 CFR § 170.110</a>)
- moreover, agencies are responsible for enforcement and
+reporting (subject to certain exceptions listed in [2 CFR §
+170.110](https://www.ecfr.gov/current/title-2/subtitle-A/chapter-I/part-170/subpart-A/section-170.110); moreover, agencies are responsible for enforcement and
```

Comment:
This edit changed the <a> tag back to a markdown link.  This needs to be changed back to an <a> tag link.

Reply:
fixed